src/test/suite/extension.test.ts

```diff
@@ -291,4 +291,12 @@ suite('utils', () => {
         assert(await exists(intoUri('../assets/org.valid.Manifest.json').fsPath))
         assert(!await exists(intoUri('../assets/sOmE.nOnExistenT.FilE.abc').fsPath))
     })
+
+    test('findFile', async () => {
```

Comment:
each `*.ts` file should probably have its own `*.test.ts` file in the `test` directory instead of having all of them in `extension.test.ts`, but out of scope of this PR.

Reply:
Yeah i agree here